azure-cli/ad/deploy.swagger.ps1

```diff
@@ -0,0 +1,80 @@
+param (
+  [Parameter(Mandatory = $true)]
+  [ValidateNotNullOrEmpty()]
+  [string]
+  $companyHostName,
+
+  [Parameter(Mandatory = $true)]
+  [EnvironmentConfig] $environmentConfig,
+
+  [Parameter(Mandatory = $true)]
+  [NamingConfig] $namingConfig,
+
+  [Parameter(Mandatory = $false)]
+  [ValidateNotNullOrEmpty()]
+  [string]
+  $serviceInstance = ""
```

Comment:
ValidateNotNullOrEmtpy, Mandatory = $false and default value "" doesn't fit together.
If parameter is optional, Parameter(Mandatory = $false)] should be enough

Reply:
Done